src/mnemonics/electrum-words.cpp

```diff
@@ -427,14 +428,14 @@ namespace crypto
 
         memwipe(w, sizeof(w));
       }
-
+      
       words += words_store[create_checksum_index(words_store, language)];
       std::time_t t = std::time(0);
       std::tm* date = std::localtime(&t);
-      size_t size = snprintf(nullptr, 0, " %4d-%02d-%02d", date->tm_year+1900, date->tm_mon+1, date->tm_mday) + 1; 
-      std::unique_ptr<char[]> buf(new char[size]); 
-      snprintf(buf.get(), size, " %4d-%02d-%02d", date->tm_year+1900, date->tm_mon+1, date->tm_mday);
-      words += std::string(buf.get(), buf.get()+size-1);
+      std::unique_ptr<char[]> buf(new char[DATE_SIZE]); 
```

Comment:
C functions are tricky, `DATE_SIZE` needs to be 10 + 1 for the null character, right?

Reply:
How about using `boost::format` instead?

Reply:
Good idea, have implemented this with boost::format now!